include/networkit/generators/BarabasiAlbertGenerator.hpp

```diff
@@ -21,7 +21,10 @@ class BarabasiAlbertGenerator final : public StaticGraphGenerator {
     count k{0};    //!< Attachments made per node
     count nMax{0}; //!< The maximal number of nodes attached
     count n0{0};   //!< The number of initial connected nodes
-    bool batagelj; //!< Specifies whether to use batagelj's method or the original one
+    bool parallel; //!< Specifies whether to use batagelj's method or a parallel one.
```

Comment:
I suggest reversing order of description (since this follows more the logic of the conditional operator):

```
Specifies whether to use a parallel algorithm or instead the algorithm of Batagelj/Brandes.
```